_includes/navbar-main.html

```diff
@@ -40,7 +41,12 @@
                             <li><a href="{{ site.baseurl }}/ome-files/">OME Files</a></li>
                         </ul>
                     </li>
-                    <li><a href="{{ site.baseurl }}/support/">Support</a></li>
+                    <li class="has-submenu"><a href="{{ site.baseurl }}/support/">Support</a>
+                        <ul class="submenu menu vertical" data-submenu>
+                            <li><a href="{{ site.baseurl }}/support/">Users</a></li>
```

Comment:
I tried not having any link at the top but without the ```<a>``` the formatting is wrong and with ```<a href="#">``` the build fails since this is not a valid link.
It's also consistent with the other about/news/products links in the nav-bar in that you can click on the main link to go to the main about or news or products page. I guess we could remove the Support link from the submenu, but a sub menu with only a single item seems kinda wrong too.

Reply:
I guess my primary concern is exactly the inconsistency with About and News links where to `/about` and `/news`  are only landing pages for all the content of the submenu. 

I do like the idea of putting `Training` alongside the support section. Would there be a way to split the content so that we have
1- a landing page
2- a `support/` page
3- a `training/` page